config/routes.rb

```diff
@@ -3,4 +3,5 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :events, only: %i[index create show update destroy]
```

Comment:
Events recource should be in `api/v1` namespace

Reply:
Added namespace `api` and `v1`